open_prices/moderation/constants.py

```diff
@@ -0,0 +1,8 @@
+REASON_OTHER = "OTHER"
```

Comment:
price
- wrong price value
- missing information (price not on proof, price/barcode not readable)
- wrong type
- other

proof
- wrong date
- wrong location
- wrong currency
- wrong type
- other

Reply:
added most of them here : https://github.com/openfoodfacts/open-prices/pull/1115/commits/96c56d87a79881463a2c95469661fe888e673b04

will leave it to another PR (and after some usage) for the model-specific rules (like the "price missing information")